templates/zerver/help/change-your-language.md

```diff
@@ -27,3 +15,14 @@ notifications language][change-org-lang].
 1. Click **Reload**.
 
 {end_tabs}
+
+!!! tip ""
+
+    To change the language of automated messages and emails in your organization, administrators
+    can [configure the organization notifications language][change-org-lang].
```

Comment:
This isn't quite right -- most emails that we send use the individual's language. The organization language feature is only used for emails we send to someone who doesn't have an account.

Reply:
I also wonder about this being a tip for an individual user looking to update their language setting UI. 

Would it be enough to have the article linked in the **Related articles** section? And that could be titled **Organization language for automated messages and invitation emails**?

Reply:
Also, wondering if we should update #22136 for that section in **Organization settings** to be more specific; e.g. **Language for automated messages and invitation emails**, instead of **Language for automated messages and emails**?

Reply:
Sounds good, I'll drop the tip. We can retitle the article in a separate pass, I think, following up on https://github.com/zulip/zulip/issues/22136. I updated the issue description.